AlwaysHoldPhysicsObjects.mlua

```diff
@@ -1,9 +1,15 @@
 --[Patch]--
 Title("Always Hold Physics Objects")
-Author("Hyper")
-Platform("Xbox 360")
+Author("Hyper & Dunker")
+Platform("All Systems")
 Blurb("Allows Silver to hold physics objects while levitating.")
 
 --[Functions]--
+Xbox 360:
 DecryptExecutable()
-WriteNopPPC(Executable|0x21DCBC)
\ No newline at end of file
+WriteNopPPC(Executable|0x21DCBC)
+
+PlayStation 3:
+DecryptExecutable()
```

Comment:
DecryptExecutable can be under an `All Systems` label before everything else - for example;
```
All Systems:
DecryptExecutable()

Xbox 360:
--360 stuff here

PlayStation 3:
--PS3 stuff here
EncryptExecutable()
```